docs/WPML-url_to_postid.trace.MD

```diff
@@ -0,0 +1,33 @@
+```
+sitepress.class.php:1176, SitePress->switch_lang()
+absolute-links.class.php:487, AbsoluteLinks->all_rewrite_rules()
+absolute-links.class.php:449, AbsoluteLinks->initialize_rewrite()
+absolute-links.class.php:100, AbsoluteLinks->_process_generic_text()
+sitepress.class.php:4270, SitePress->url_to_postid()
+class-wp-hook.php:307, WP_Hook->apply_filters()
+plugin.php:189, apply_filters()
+rewrite.php:498, url_to_postid()
+embed.php:662, get_oembed_response_data_for_url()
+embed.php:1231, wp_filter_pre_oembed_result()
+class-wp-hook.php:307, WP_Hook->apply_filters()
+plugin.php:189, apply_filters()
+class-wp-oembed.php:395, WP_oEmbed->get_html()
+embed.php:115, wp_oembed_get()
+class-wp-embed.php:305, WP_Embed->shortcode()
+class-wp-embed.php:463, WP_Embed->autoembed_callback()
+class-wp-embed.php:445, preg_replace_callback()
+class-wp-embed.php:445, WP_Embed->autoembed()
+class-wp-hook.php:307, WP_Hook->apply_filters()
+plugin.php:189, apply_filters()
+class-wp-rest-revisions-controller.php:606, WP_REST_Revisions_Controller->prepare_item_for_response()
+class-wp-rest-autosaves-controller.php:408, WP_REST_Autosaves_Controller->prepare_item_for_response()
+class-wp-rest-autosaves-controller.php:300, WP_REST_Autosaves_Controller->get_items()
+class-wp-rest-server.php:1141, WP_REST_Server->respond_to_request()
+class-wp-rest-server.php:988, WP_REST_Server->dispatch()
+rest-api.php:511, rest_do_request()
+rest-api.php:2860, rest_preload_api_request()
+block-editor.php:501, array_reduce()
+block-editor.php:501, block_editor_rest_api_preload()
```

Comment:
Problem 1: WP's has an "optimization" to preload post revisions before sending the editor HTML. It renders the content of every revision, regardless of how many total.